cmd/notify-mailer/main.go

```diff
@@ -108,119 +109,141 @@ func (m *mailer) run() error {
 		return err
 	}
 
-	m.log.Infof("Resolving %d destination addresses", len(m.destinations))
 	addressesToRecipients, err := m.resolveEmailAddresses()
 	if err != nil {
 		return err
 	}
-	if len(addressesToRecipients) == 0 {
-		return fmt.Errorf("zero recipients after looking up addresses?")
+
+	totalAddresses := len(addressesToRecipients)
+	if totalAddresses == 0 {
+		return errors.New("0 recipients remained after resolving addresses")
 	}
-	m.log.Infof("Resolved destination addresses. %d accounts became %d addresses.",
-		len(m.destinations), len(addressesToRecipients))
-	var biggest int
-	var biggestAddress string
-	for k, v := range addressesToRecipients {
-		if len(v) > biggest {
-			biggest = len(v)
-			biggestAddress = k
+
+	m.log.Infof(
+		"%d recipients were resolved to %d addresses",
+		len(m.recipients), totalAddresses)
```

Comment:
Indentation style here does not match style on line 94, despite line 94's format string being longer.